clients/web/src/stylesheets/body/userList.styl

```diff
@@ -58,3 +58,22 @@
     input[type="text"]
       width 170px
       margin-right 13px
+
+  .g-user-count-container
+    display inline-block
+    color #888
+    font-size 16px
+
+    i.icon-user
```

Comment:
I wouldn't apply the `.icon-user` selector here, so the position doesn't break if we ever change the specific icon that's used. The fact that it's an `i` element inside a `.g-user-count-container` is specific enough.